.github/skills/sync-ai-skills/scripts/sync_ai_skills.sh

```
@@ -0,0 +1,139 @@
+    printf 'Verification failed: missing destination skill directory: %s\n' \
+      "$target_skill_dir" >&2
+    exit 1
+  fi
+
+  while IFS= read -r source_file; do
+    local relative_path
+    local target_file
+    local source_hash
+    local target_hash
+
+    relative_path="${source_file#$source_skill_dir/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)